zulip_bots/generate_manifest.py

```diff
@@ -42,17 +44,18 @@ def get_assets():
     )
     return assets_files
 
-def main():
+def generate_dev_manifest():
     # type: () -> None
-    manifest_path = os.path.join(CURRENT_DIR, 'MANIFEST.in')
-    with open(manifest_path, 'w') as fp:
+    with open(MANIFEST_PATH, 'w') as fp:
         template = 'include {line}\n'
         fixtures = map(lambda line: template.format(line=line),
                        get_test_fixtures())
         logos = map(lambda line: template.format(line=line),
                     get_logos())
         docs = map(lambda line: template.format(line=line),
                    get_docs())
+        assets = map(lambda line: template.format(line=line),
+                     get_assets())
```

Comment:
Hmm, not sure, could be! But I wanted to avoid the functional changes that separated two MANIFEST files from the changes that are independent of that separation. Does that make any sense?